.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.19.0.cjs"
```

Comment:
If yarn is greater than 1.19 locally, it throws up with weird errors when you do `yarn add` inside a workspace.

Their docs mention we shouldn't gitignore `.yarnrc` 
https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored